data/json/items/armor/ballistic_armor.json

```diff
@@ -209,12 +219,13 @@
     "name": { "str": "scrap ESAPI plate" },
     "//": "some crude metal chopped and bent to fit into a plate carrier. not great, but better than nothing. added the cotton component cause i'd imagine a jagged piece of metal would cut through a ballistic vest and this is based on the existing scrap armor",
     "//2": "requested in https://github.com/CleverRaven/Cataclysm-DDA/issues/49477",
-    "description": "A flat piece of salvaged steel, crudely cut and bent into the shape of an ESAPI ballistic plate.  It's wrapped in rags to cover the sharp edges.  Better than nothing, but it's heavy, and not very strong.  This is intended to be inserted into a ballistic vest, and can't be worn on its own.",
-    "weight": "900 g",
-    "//3": "30cm x 25cm x 1.5cm",
-    "volume": "188 ml",
+    "description": "A flat piece of salvaged steel, crudely cut and bent into the shape of an ESAPI ballistic plate.  It's wrapped in rags to cover the sharp edges.  Better than nothing, but it's heavier than the real thing, and not very strong.  This is intended to be inserted into a ballistic vest, and can't be worn on its own.",
+    "weight": "4535 g",
+    "//3": "30cm x 25cm x 2cm",
```

Comment:
this should be .15cm and you would have changed it to .2cm that's a typo in the comment. Though this begs the question, where is the weight coming from? steel isn't magic why are these heavier than the spartan plates you referenced above for a thinner plate?

Reply:
These plates @ 2mm give almost the same amount of ballistic protection as a leather jacket, but this PR is just for fixing the weight and volume so I'll leave adjustments for another one.